src/main/java/core/basesyntax/dao/fileReader.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.dao;
+
+public interface fileReader {
```

Comment:
The interface name `fileReader` does not follow Java naming conventions. Interface names should be in PascalCase, such as `FileReader`.